docker/docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: '2'
+
+services:
+  celery:
+    build:
+      context: ..
+      dockerfile: docker/Dockerfile
+    environment:
+      TEST_BROKER: pyamqp://rabbit:5672
+      TEST_BACKEND: redis://redis
+      PYTHONUNBUFFERED: 1
+      PYTHONDONTWRITEBYTECODE: 1
+      REDIS_HOST: redis
+      WORKER_LOGLEVEL: INFO
+    tty: true
+    volumes:
+      - ../celery:/home/someone/celery
+      # Because of the __pycache__ files, PYTHONDONTWRITEBYTECODE must be
+      # set on the host as well or py.test will throw configuration errors.
+      - ../t:/home/someone/t
+    depends_on:
+      - rabbit
+      - redis
+      - dynamodb
+
+  rabbit:
```

Comment:
Generally it is recommended to use exact tags instead of latest images, as it improves consistency and we can always ask on bug reports which version of a database someone uses.

Reply:
Good point -- will do.